Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -460,7 +539,7 @@ void OrientSelf()
 		if ( IsInState( ActorStates.Climbing ) )
 		{
 			Vector3 lookVector = _climbSurfaceNormal;
-			//lookVector.y *= _leanTowardsSurface;
+			lookVector.y *= _leanTowardsSurface;
```

Comment:
Double check for warnings. I was getting an unused variable warning last night. This looked like it fixed it but double check for any others.